tsconfig.json

```diff
@@ -25,10 +25,11 @@
   "exclude": [
     "node_modules",
     "build",
-    "tst",
+    "demo",
```

Comment:
Is this "demo" under the integration folder? We do not have a demo in the root folder right?

Reply:
Not sure why it is showing up as an addition here but this `demo` has been in the config for a while it seems. https://github.com/aws/amazon-chime-sdk-component-library-react/blob/8ea667fb87a1e29d73d04008707775b8aadef941/tsconfig.json#L28

There is a demo folder under root folder.

Reply:
I removed this in my PR. We don't have demo folder now.
You don't need to add it back, due to no harm.